android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -385,6 +382,20 @@ private void downloadPageId() {
                     });
     }
 
+    private void storePageName(String feedType, String key) {
+        if(SharedPreferencesUtil.getString(key, null) == null) {
+            RealmList<SocialLink> socialLinks = event.getSocialLinks();
```

Comment:
Tested it. It is giving error because of even object being null. Please check again